evaluate.py

```diff
@@ -78,6 +78,8 @@ def main(
         if not isinstance(dt, torch.dtype):
             raise ValueError(f"{dtype} is not a valid dtype.")
         dtype = dt
+    else:
+        dtype = torch.float32
```

Comment:
If you change `dtype` to `: str = "float32` in the function definition, we could get rid of this `if/else` and also make it clearer what's the default

This comment applies to the other files too

Reply:
Done!